src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public List<String> generateReport() {
+        return Storage.fruits.values().stream()
```

Comment:
Ensure that the `Storage.fruits` field is properly encapsulated. Direct access to the field from outside the `Storage` class violates the encapsulation principle. Instead, you should provide a method in `Storage` that returns the necessary data.